src/Docker/Manager/ContainerManager.php

```diff
@@ -172,6 +172,32 @@ public function start(Container $container, array $hostConfig = [])
         return $this;
     }
 
+
+    /**
+     * Copy files or folders from a container
+     *
+     * @param \Docker\Container $container
+     * @param                   $resource file or folder name
```

Comment:
Missing variable type here, I assume it would be `string`? Maybe you could rename the argument too, because `$resource` is misleading if it can't be an actual resource. What about `$source`?